plugin/discovery/find.go

```diff
@@ -96,6 +96,7 @@ func findPluginPaths(kind string, machineName string, dirs []string) []string {
 				continue
 			}
 
+			// FIXME: we pass in GOOS_GOARCH paths directly, so these may not be "legacy"
```

Comment:
yeah, the FIXME probably isn't needed here; it's on my own list, and the log output shows the legacy warning too.